README.adoc

```diff
@@ -0,0 +1,11 @@
+= secret-operator
+
+Manages/generates Kubernetes secrets, and mounts them into pods using CSI
+
+== Running
+
+The operator must be executed inside Kubernetes, since it registers a CSI plugin against Kubelet.
+
+Currently builds are only supported with Nix. To build and install, run:
+
+    nix run -f . crate2nix generate && nix build -f . docker && kind load image-archive <(./result) && kubectl apply -f provisioner.yaml && kubectl rollout restart ds/secret-provisioner
```

Comment:
For me and a fresh nix install:
`nix run -f . crate2nix generate --experimental-features nix-command && nix build -f . docker --experimental-features nix-command && kind load image-archive <(./result) && kubectl apply -f provisioner.yaml && kubectl rollout restart ds/secret-provisioner`

Reply:
6d072cc9dd5cbd368953d65b572f72c5b65e1acf